generic/MAPL_Generic.F90

```diff
@@ -11334,4 +11339,43 @@ subroutine warn_empty(string, MPL, rc)
       _RETURN(ESMF_SUCCESS)
    end subroutine warn_empty
 
+   recursive subroutine MAPL_AddI4AttributeToFields(gc,field_name,att_name,att_val,rc)
```

Comment:
```suggestion
   recursive subroutine MAPL_AddAttributeToFields_I4
   (gc,field_name,att_name,att_val,rc)
```
Nominal preference to put the disambguators at the end to make them easier to spot.  I think this is more consistent with other cases, though I'm sure it is far from uniform in MAPL.  No need to change if this is the only issue I find.

Reply:
Made the change